docs/README.md

```diff
@@ -0,0 +1,63 @@
+# PyTorch-Lightning Docs
+
+We are using Sphinx with Napoleon extension.
+Moreover, we set Google style to follow with type convention.
+
+- [Napoleon formatting with Google style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)
+- [ReStructured Text (reST)](https://docs.pylonsproject.org/projects/docs-style-guide/)
+- [Paragraph-level markup](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#paragraphs)
+
+See following short example of a sample function taking one position string and optional
+
+```python
+from typing import Optional
+
+
+def my_func(param_a: int, param_b: Optional[float] = None) -> str:
+    """Sample function.
+
+    Args:
+        param_a: first parameter
+        param_b: second parameter
+
+    Return:
+        sum of both numbers
+
+    Example::
+
+        Sample doctest example...
+        >>> my_func(1, 2)
+        3
+
+    Note:
+        If you want to add something.
+    """
+    p = param_b if param_b else 0
+    return str(param_a + p)
+```
+
+## Running tests
```

Comment:
Does this mislead to us having some tests for docs? Should this heading be something else?

Reply:
`Building docs`?